config/components/visibility/prioritylevelconfigurations.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta3
+kind: PriorityLevelConfiguration
+metadata:
+  name: visibility
+spec:
+  limited:
+    lendablePercent: 0
```

Comment:
As suggested in https://github.com/kubernetes-sigs/kueue/pull/3032#discussion_r1760608579, I would suggest to increase the limit to 90

Reply:
cc @tenzen-y

Reply:
@mbobrovskyi @mimowo Is there any progressing for this?

Reply:
Done